src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -436,6 +436,13 @@ public async Task FireReminderAsync(ActorReminder reminder)
 
             try
             {
+                ActorTrace.Source.WriteInfoWithId(

```

Comment:
- Emit a log before acquiring the actor lock
- Emit a log after acquiring the actor lock 
(Basically add traces for the lock events which are going into diagnostics events). 
This will help us answer the queries around "My reminder did not fire on time" related questions.